tests/modules/users/resources/test_options.py

```diff
@@ -14,9 +14,10 @@
     ),
 )
 def test_users_options_unauthorized(
-    path, status_code, expected_allowed_methods, flask_app_client
+    path, status_code, expected_allowed_methods, flask_app_client, admin_user
 ):
-    response = flask_app_client.options(path)
+    with flask_app_client.login(admin_user, auth_scopes=('users:write',)):
```

Comment:
Failed attempt to fix the current user is absent issue, will be reverted in next commit.